tempo/noauth/tempo.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: project.openshift.io/v1
+kind: Project
+metadata:
+  name: tempo-example
+---
+# This secret is needed to indicate Tempo how to connect to the minio instance
+apiVersion: v1
+kind: Secret
+metadata:
+  name: minio-test
```

Comment:
I'd go with `minio` instead of `minio-test` for this example, which afaics should be our recommended approach of deploying this in production.